addons-l10n/en/name-backpack-items.json

```
@@ -1,3 +1,4 @@
 {
+  "name-backpack-items/title": "Put script into backpack",
```

```suggestion
  "name-backpack-items/title": "Add to Backpack",
```
Scratch's own modals have short titles and use title case.